From 4316541b079a6621d47070a27dd66c7ba9103d45 Mon Sep 17 00:00:00 2001
From: Harinarayan Bhatta <hbhatta@nvidia.com>
Date: Wed, 22 Jul 2020 13:15:06 +0530
Subject: [PATCH] no atexit in trusty

---
 crypto/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/init.c b/crypto/init.c
index 1b0d523..2ce800c 100644
--- a/crypto/init.c
+++ b/crypto/init.c
@@ -131,7 +131,7 @@ DEFINE_RUN_ONCE_STATIC(ossl_init_register_atexit)
 #ifdef OPENSSL_INIT_DEBUG
     fprintf(stderr, "OPENSSL_INIT: ossl_init_register_atexit()\n");
 #endif
-#ifndef OPENSSL_SYS_UEFI
+#if !defined(OPENSSL_SYS_UEFI) && !defined(OPENSSL_SYS_TRUSTY)
 # ifdef _WIN32
     /* We use _onexit() in preference because it gets called on DLL unload */
     if (_onexit(win32atexit) == NULL)
-- 
2.7.4

